scripts/db_copy_to_localhost.sh

```
@@ -13,29 +13,28 @@ if [[ ${ENV} != staging ]] && [[ ${ENV} != prod ]]; then
+FILENAME="`date +"%Y-%m-%d"`-${ENV}.pgsql"
+
+if [[ ! -d ${DBDUMPS_DIR} ]]; then
+  mkdir -p "${DBDUMPS_DIR}"
+fi
+
+if [[ ! -s ${DBDUMPS_DIR}${FILENAME} ]]; then
+  echo "Dumping ${ENV} database"
+  if [ -x "$(command -v heroku)" ]; then
+    heroku pg:backups:download -a "opencollective-${ENV}-api" -o "${DBDUMPS_DIR}${FILENAME}"
+  else
+    PG_URL_ENVIRONMENT_VARIABLE=`heroku config:get PG_URL_ENVIRONMENT_VARIABLE -a "opencollective-${ENV}-api"`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)